packages/backend/src/error/user/user-not-found-error.ts

```diff
@@ -1,11 +1,10 @@
 import { HttpError, HttpStatusCode } from '@vse-bude/shared';
-import type { Request } from 'express';
 
 class UserNotFoundError extends HttpError {
-  constructor(req: Request) {
+  constructor() {
     super({
       status: HttpStatusCode.NOT_FOUND,
-      message: req.t('USER_NOT_FOUND'),
+      message: 'User not found!',
```

Comment:
why not localised?

Reply:
cause we decided with Vitalli to refactor a little bit, because it's really hard to inject express Request in undefined amount of constructors to translate some text